tests/tests.py

```diff
@@ -943,6 +944,11 @@ def test_plantcv_parallel_process_results_invalid_json():
 TEST_THERMAL_ARRAY = "thermal_img.npz"
 TEST_THERMAL_IMG_MASK = "thermal_img_mask.png"
 TEST_INPUT_THERMAL_CSV = "FLIR2600.csv"
+TEST_BAD_MASK = "bad_mask_test.pkl"
+TEST_IM_BAD_NONE = "bad_mask_none.pkl"
+TEST_IM_BAD_BOTH = "bad_mask_both.pkl"
+TEST_IM_BAD_NAN = "bad_mask_nan.pkl"
+TEST_IM_BAD_INF = "bad_mask_inf.pkl"
```

Comment:
Couldn't we just have the masks in the data folder instead of pickle files?

Reply:
Did you mean save the masks as grayscale images?